kernel/configs/fragments/whitelist.conf

```diff
@@ -0,0 +1,11 @@
+# configuration options which may dropped in newer kernels
+# without generating an error in fragment merging
+CONFIG_INET6_XFRM_MODE_BEET
```

Comment:
As @mythi spotted in https://github.com/kata-containers/packaging/pull/698/files, shouldn't all these options be set to `=y`?

Reply:
These are config strings, not settings. This just means to ignore errors when doing merge for these particular items when using experiments kernel.

Reply:
the side effect is that if another fragment tries to override this setting, `.config` generation fails with an error.